### ISSUE TITLE
Kandinsky 3.0 is very slow

### ISSUE THREAD

**Issue by dextercry** [seruva19/kubin#167]
On my 3090 Kandinsky 2.2 generates images with 6 it/s
on Kandinsky 3.0 I get like 300+ seconds / it

I think it goes only with CPU 

Here is a log from when I started 

A matching Triton is not available, some optimizations will not be enabled.
Error caught was: No module named 'triton'
command line arguments: {}
root dir: D:\kubin
found 11 extensions
1: extension 'kd-upscaler' found
1: extension 'kd-upscaler' installation integrity verified
1: extension 'kd-upscaler' successfully registered
2: extension 'kd-animation' found
2: extension 'kd-animation' installation integrity verified
2: extension 'kd-animation' successfully registered
3: extension 'kd-image-to-video' found
3: extension 'kd-image-to-video' installation integrity verified
3: extension 'kd-image-to-video' successfully registered
4: extension 'kd-interrogator' found
4: extension 'kd-interrogator' installation integrity verified
4: extension 'kd-interrogator' successfully registered
5: extension 'kd-mesh-gen' found
5: extension 'kd-mesh-gen' installation integrity verified
5: extension 'kd-mesh-gen' successfully registered
6: extension 'kd-multi-view' found
6: extension 'kd-multi-view' installation integrity verified
6: extension 'kd-multi-view' successfully registered
7: extension 'kd-prompt-styles' found
7: extension 'kd-prompt-styles' successfully registered
8: extension 'kd-segmentation' found
8: extension 'kd-segmentation' installation integrity verified
8: extension 'kd-segmentation' successfully registered
9: extension 'kd-image-browser' found
9: extension 'kd-image-browser' successfully registered
10: extension 'kd-networks' found
10: extension 'kd-networks' installation integrity verified
10: extension 'kd-networks' successfully registered
11: extension 'kd-training' found
11: extension 'kd-training' installation integrity verified
11: extension 'kd-training' successfully registered
extension post-install phase: verified
activating pipeline: diffusers (3.0)
scanning system information
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
scanning system information
applying changes: {'values_changed': {"root['general']['model_name']": {'new_value': 'kd22', 'old_value': 'kd30'}}}
model will be reloaded because of config changes
flush: not implemented
activating pipeline: diffusers (2.2)
task queued: text2img
Loading pipeline components...: 100%|████████████████████████████████████████████████████| 6/6 [00:06<00:00,  1.14s/it]
following pipelines, if active, will be released for text2img task: ['text2img_cnet', 'inpainting']
Loading pipeline components...: 100%|████████████████████████████████████████████████████| 3/3 [00:00<00:00,  6.78it/s]
optimizations: none
seed generated: 33844911125
100%|██████████████████████████████████████████████████████████████████████████████████| 25/25 [00:00<00:00, 35.93it/s]
prior embeddings: done
negative prior embeddings: done
using sampler: DDPM
100%|██████████████████████████████████████████████████████████████████████████████████| 50/50 [00:08<00:00,  6.17it/s]
decoder images: done
task queued: text2img
optimizations: none
seed generated: 22592274418
100%|██████████████████████████████████████████████████████████████████████████████████| 25/25 [00:00<00:00, 45.54it/s]
prior embeddings: done
negative prior embeddings: done
using sampler: DDIM
100%|██████████████████████████████████████████████████████████████████████████████████| 50/50 [00:08<00:00,  5.73it/s]
decoder images: done
task queued: text2img
optimizations: none
seed generated: 80290159821
100%|██████████████████████████████████████████████████████████████████████████████████| 25/25 [00:00<00:00, 45.17it/s]
decoder images: done
applying changes: {'values_changed': {"root['general']['model_name']": {'new_value': 'kd30', 'old_value': 'kd22'}}}
model will be reloaded because of config changes
following pipelines, if active, will be released: ['text2img', 'img2img', 'mix', 'inpainting', 'outpainting', 'text2img_cnet', 'img2img_cnet', 'mix_cnet']
releasing t2i pipeline
Pipelines loaded with `dtype=torch.float16` cannot run with `cpu` device. It is not recommended to move them to `cpu` as running them will fail. Please make sure to use an accelerator to run the pipeline in inference, due to the lack of support for`float16` operations on this device in PyTorch. Please, remove the `torch_dtype=torch.float16` argument, or use another device for inference.
Pipelines loaded with `dtype=torch.float16` cannot run with `cpu` device. It is not recommended to move them to `cpu` as running them will fail. Please make sure to use an accelerator to run the pipeline in inference, due to the lack of support for`float16` operations on this device in PyTorch. Please, remove the `torch_dtype=torch.float16` argument, or use another device for inference.
activating pipeline: diffusers (3.0)
task queued: text2img
flush: not implemented
Loading pipeline components...:  40%|████████████████████▊                               | 2/5 [00:00<00:00,  3.66it/s]Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
Loading checkpoint shards: 100%|█████████████████████████████████████████████████████████| 5/5 [00:00<00:00,  5.66it/s]
Loading pipeline components...: 100%|████████████████████████████████████████████████████| 5/5 [00:01<00:00,  2.84it/s]
seed generated: 81470437690
 20%|████████████████▌                                                                  | 1/5 [05:48<23:14, 348.58s/it]

**Comment by seruva19** [comment]
Kandinsky 3 text encoder is very large, it does not fit even into 24 Gb VRAM. And I have no idea how to deal with it so far.
But you cant try enabling sequential offloading like described here https://github.com/seruva19/kubin/wiki/Docs#kandinsky-3, it should run faster.
